File: ulyp-ui/src/main/java/com/ulyp/ui/util/ClassNameUtils.java
package com.ulyp.ui.util;

public class ClassNameUtils {

    public static String toSimpleName(String name) {
        int lastDot = name.lastIndexOf('.');
        if (lastDot > 0) {
            return name.substring(lastDot + 1);
        } else {
            return name;
        }
    }
}

File: ulyp-ui/src/main/kotlin/com/ulyp/ui/util/ClassNameUtils.kt
package com.ulyp.ui.util

object ClassNameUtils {
    @JvmStatic
    fun toSimpleName(name: String): String {
        val lastDot = name.lastIndexOf('.')
        return if (lastDot > 0) {
            name.substring(lastDot + 1)
        } else {
            name
        }
    }
}